cmd/workload-controller-manager/config/controllerconfig.json

```diff
@@ -2,15 +2,18 @@
     "controllers": [
         {
             "type":    "node",
-            "workers": 5
+            "workers": 5,
+            "reportHealthIntervalInSecond":  20
```

Comment:
Heath report interval shall be consistent. Please keep a single value and don't duplicate necessary

Reply:
Perhaps do 10 second per report as API server renew lease every 10 seconds.